### PR TITLE
Remove unnecessary environment variables

### PR DESCRIPTION
**Pull request by kazyshr**

### CODE REVIEW

File: ecl/provider_test.go [hunk 1]
```diff
@@ -18,41 +18,16 @@ import (
...
 	OS_INTERNET_SERVICE_ZONE_NAME            = os.Getenv("OS_INTERNET_SERVICE_ZONE_NAME")
-	OS_LB_ENVIRONMENT                        = os.Getenv("OS_LB_ENVIRONMENT")
-	OS_MAIL_ADDRESS                          = os.Getenv("OS_MAIL_ADDRESS")
-	OS_MAGNUM_FLAVOR                         = os.Getenv("OS_MAGNUM_FLAVOR")
-	OS_NETWORK_ID                            = os.Getenv("OS_NETWORK_ID")
-	OS_POOL_NAME                             = os.Getenv("OS_POOL_NAME")
 	OS_QOS_OPTION_ID_100M                    = os.Getenv("OS_QOS_OPTION_ID_100M")
 	OS_QOS_OPTION_ID_10M                     = os.Getenv("OS_QOS_OPTION_ID_10M")
 	OS_REGION_NAME                           = os.Getenv("OS_REGION_NAME")
-	OS_FORCE_SSS_ENDPOINT                    = os.Getenv("OS_FORCE_SSS_ENDPOINT")
```
Comment (ManabuMiwa) [inline ecl/provider_test.go]: It looks to be marked as "要" in the spread sheet:
https://docs.google.com/spreadsheets/d/1nlQg8PLBBbl8_W55Y69cHquVld-aw7s3heCRRBir1PU/edit#gid=2105055935

File: ecl/provider_test.go [hunk 2]
```diff
@@ -18,41 +18,16 @@ import (
...
-	OS_SSS_TENANT_ENVIRONMENT                = os.Getenv("OS_SSS_TENANT_ENVIRONMENT")
-	OS_SSS_USER_ENVIRONMENT                  = os.Getenv("OS_SSS_USER_ENVIRONMENT")
-	OS_STORAGE_VOLUME_TYPE_ID                = os.Getenv("OS_STORAGE_VOLUME_TYPE_ID")
-	OS_SUBNET_ID                             = os.Getenv("OS_SUBNET_ID")
-	OS_SWIFT_ENVIRONMENT                     = os.Getenv("OS_SWIFT_ENVIRONMENT")
 	OS_TENANT_ID                             = os.Getenv("OS_TENANT_ID")
-	OS_TENANT_NAME                           = os.Getenv("OS_TENANT_NAME")
```
Comment (ManabuMiwa) [inline ecl/provider_test.go]: It looks to be marked as "要" in the spread sheet:
https://docs.google.com/spreadsheets/d/1nlQg8PLBBbl8_W55Y69cHquVld-aw7s3heCRRBir1PU/edit#gid=2105055935

File: ecl/provider_test.go (hunk 1 above)
Comment (kazyshr) [reply ecl/provider_test.go]: I did not remove an environment variable `OS_FORCE_SSS_ENDPOINT` but a variable in test code. This variable was declared for test codes but was not used anywhere. That's why I removed it.

File: ecl/provider_test.go (hunk 2 above)
Comment (kazyshr) [reply ecl/provider_test.go]: Please look at another comment. Actually, I did not remove environment variable.

**Pull request merged**